test/SimpleExchange.js

```diff
@@ -229,14 +229,59 @@ contract(
       expect(rateAfterFailedTx[0]).to.be.bignumber.eq(Q18.mul(100));
     });
 
-    it("should revert on set exchange rate when setting ", async () => {
+    it("should revert on set exchange rate when setting same nominator and denominator", async () => {
       await expect(
         gasExchange.setExchangeRate(etherToken.address, etherToken.address, Q18.mul(0.1), {
           from: tokenOracleManager,
         }),
       ).to.be.rejectedWith("SEX_SAME_N_D");
     });
 
+    it("should revert when trying to set rate for addres that is not erc223 token", async () => {
+      await expect(
+        gasExchange.setExchangeRate(etherToken.address, randomAddress, Q18.mul(1), {
+          from: tokenOracleManager,
+        }),
+      ).to.revert;
+    });
+
+    it("should revert when inversed rate is more than uint256 max, but because after inversion it equal to zero", async () => {
```

Comment:
two options here
1. we remove the test
2. we test that biggest possible inverse is 10**36 (without even calling contract)